### ISSUE TITLE
free logo gesign

### ISSUE THREAD

**Issue by Pelikens** [manastalukdar/learning-computer-science#43]
## Other notes
Hi, my name is Ksenia. I'm a graphic designer and I'd like to collaborate on your open source project and propose free logo/icon design If it's something you're interested in, please, let me know!
Best,
Ksenia

**Comment by manastalukdar** [comment]
You are quite welcome to propose a logo. The best way to go about it is to fork the project, add the image in the resources/images directory in your fork, then submit a PR.

Before you do that, some screenshots of what you have in mind, would be good.

Just be aware that this project is my own personal learning project. I have a website that acts a search engine interface for the data in the repo, but it is yet unadvertised. As you may have found, the readme.md at the root is also very bare-bones.

**Comment by Pelikens** [comment]
Do you have any special requirements for the logo? Any symbols that should be included?  Colors?

**Comment by manastalukdar** [comment]
I would prefer

- something minimalist
- nothing too flashy
- simple yet something that by being unique
  - catches the eye
  - is retained in memory 

Those would be the only requirements from my side. Thanks for offering to do this.

**Comment by Pelikens** [comment]
ok, what name do you want to write? learning-computer-science?

**Comment by manastalukdar** [comment]
Yes, that would be fine, without the hyphens. I am also open to alternatives. Do you have any suggestions?

**Comment by Pelikens** [comment]
May be initials LCS would be better? I'll try both make options.

**Comment by manastalukdar** [comment]
LCS sounds like some algorithm. Yes, if you have time, I think taking a look at both options will help decide.

**Comment by Pelikens** [comment]
Hi, here is an option for the logo. Please, let me know what do you think?

![image](https://user-images.githubusercontent.com/39441298/45138489-43d48c00-b1b5-11e8-80de-dc8b4ee74854.png)

**Comment by manastalukdar** [comment]
Looks good.

**Comment by Pelikens** [comment]
If all approve, i'll open the PR when I make you options of all sizes. color and monochrome, OK?

**Comment by manastalukdar** [comment]
Sounds good. Please also provide the raw files used to create the final images. Just curious, what software do you use?

**Comment by Pelikens** [comment]
I work in Adobe Illustrator. For you I'll made AI, EPS and PNG, PSD, SVG. Ok?

**Comment by manastalukdar** [comment]
Perfect.
